Misc/NEWS.d/next/Library/2023-09-08-19-44-01.gh-issue-109151.GkzkQu.rst

```diff
@@ -0,0 +1 @@
+Enable ``readline`` editing feasibilities in the :mod:`sqlite3` CLI.
```

Comment:
English is not my first language, but "editing feasibilities" sounds surprising to me:

```suggestion
Enable ``readline`` editing features in the :ref:`sqlite3 command-line interface <sqlite3-cli>` (``python -m sqlite3``).
```